assembl/static2/css/components/posts.scss

```diff
@@ -240,17 +280,13 @@ $deleted-post-border: 1px solid $light-grey !default;
     }
   }
 
-  .translate {
-    padding: 20px 0 0;
-  }
-
   .box {
     border: none;
     margin: 0 auto;
-    max-width: 600px;
+    max-width: 925px;
```

Comment:
what's that max-width ? is there any relation with responsive break point ?

Reply:
Yes, the breaking point is equal to the width of the post. If not, the browser windows will crop the post